internal/k8s/controller_test.go

```diff
@@ -8,9 +8,9 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/nginxinc/kubernetes-ingress/internal/configs"
 	"github.com/nginxinc/kubernetes-ingress/internal/nginx"
-	"github.com/nginxinc/kubernetes-ingress/internal/nginx/plus"
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
```

Comment:
What version of go did you use for this? My `go-fmt` still adds `v1` automatically here 🤔

Reply:
Mine doesn't anyway - `go version go1.12 darwin/amd64`

Reply:
Weird